src/ontology/uberon.Makefile

```diff
@@ -166,12 +157,6 @@ uberon.owl: $(OWLSRC) $(BRIDGEDIR)/uberon-bridge-to-bfo.owl $(DEVELOPS_FROM_CHAI
 	         unmerge -i $(DEVELOPS_FROM_CHAIN) \
 	         annotate -O $(URIBASE)/$@ -V $(RELEASE)/$@ -o $@
 
-# Step 2.1: Generating other formats.
-# also do OE check here
-uberon.obo: uberon.owl
-	$(MAKEOBO)
-.PRECIOUS: uberon.obo
```

Comment:
Have you tried this? This is basically switching the uberon.obo release (one of teh most important release files) to ODK default right? Does it still look ok in the diff?

Reply:
Yes, I tried. The diff is (expectedly) quite large, because one of the things the custom `MAKEOBO` rule did was to remove _all_ the `property_value:` tags. With the standard ODK rule, all those tags are still there.

Whether this is a problem or not, I don’t know. But if it is not a problem for all the ontologies using the standard ODK rule, why would that be a problem for Uberon?

Reply:
I suspect this could be a problem only if a downstream user is parsing the `uberon.obo` file with some weird or application-specific script (maybe a script that doesn’t even know about the existence of the `property_value` tag?). But if so, that’s the kind of things we may very well never know about.

Reply:
Ok, I can get behind that! Thanks for checking.

Reply:
If someone does need a OBO version of Uberon that is stripped of those tags, I’d suggest to do the same thing we do in FlyBase. The OBO parser we use to load our ontologies into the database does have some special constraints, so we have a special release artefact to produce an OBO file that meet those constraints.

We could do the same here: leave the `uberon.obo` file be the file generated by the standard ODK rule, and have an extra `uberon-uncomplicated-obo-version.obo` where we strip everything that a (too) simple parser might not like. But we should only do that if, and only if, someone asks for it.

Reply:
ok!